src/applications/edu-benefits/10203/tests/schema/confirmation-stem-test.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
What is the difference between this and `e2e-maximal-test.json` or `maximal-json` or `minimal-json`?

If not using `e2e-maximal-test.json` it should be removed

Reply:
Good point, we should probably just delete all of the old e2e data.